### PR TITLE
Conversation

### PR DESCRIPTION
**Pull request by Jhovannycanaspino**
## Summary
- Completed the conveesation

## Screenshot
image

## Trello Card
https://trello.com/c/ibE0fqm3

### CODE REVIEW

File: reactor-training/src/main/java/com/wolox/reactortraining/facade/BotFacade.java [hunk 1]
```diff
@@ -39,4 +42,19 @@ public void createBot(BotRequest botRequest) throws Exception {
   public Flux<String> getTwettstreaming(BotRequest botRequest) throws Exception {
     return this.twitterService.getTweetStream(botRequest);
   }
+
+  public Flux<String> getConversation(List<String> bot) {
+    Random random = new Random();
+    Flux<String> botUno = Flux.just(bot.get(0)).repeatWhen(r -> Flux.range(1, 10)).map( s ->
```
Comment (darkBasic666) [inline reactor-training/src/main/java/com/wolox/reactortraining/facade/BotFacade.java]: if you recieve 3 bots ?

File: reactor-training/src/main/java/com/wolox/reactortraining/facade/BotFacade.java [hunk 2]
```diff
@@ -39,4 +42,19 @@ public void createBot(BotRequest botRequest) throws Exception {
...
+        this.botService.getBotTalk(s, "200")
+    ).take(random.nextInt(7 )+ 3).flatMap(botResponseMono -> botResponseMono.map(botResponse -> botResponse.getName() + botResponse.getResponse()));
```
Comment (darkBasic666) [inline reactor-training/src/main/java/com/wolox/reactortraining/facade/BotFacade.java]: Add a better separator. ie : Wolox :

File: reactor-training/src/main/java/com/wolox/reactortraining/facade/BotFacade.java [hunk 3]
```diff
@@ -39,4 +42,19 @@ public void createBot(BotRequest botRequest) throws Exception {
...
+
+    Flux<String> botDos = Flux.just(bot.get(1)).repeatWhen(r -> Flux.range(1, 10)).map( s ->
+        this.botService.getBotTalk(s, "200")
```
Comment (darkBasic666) [inline reactor-training/src/main/java/com/wolox/reactortraining/facade/BotFacade.java]: why send a string ? 
size of length of bot talk , must to by ramdom too.

**Review by darkBasic666 — CHANGES_REQUESTED** [review]

**Comment by darkBasic666** [comment]
Excellent work using streaming data.

**Review by yanzacd — APPROVED** [review]

**Review by darkBasic666 — APPROVED** [review]

**Pull request merged**